src/components/Text/Text.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from 'react';
+import classNames from 'classnames';
+import styles from './Text.module.scss';
+
+export type TextProps = {
+  /** Дополнительный класс */
+  className?: string;
+  /** Стиль отображения */
+  view?: 'title' | 'button' | 'p-20' | 'p-18' | 'p-16' | 'p-14';
+  /** Html-тег */
+  tag?: 'h1' | 'h2' | 'h3' | 'h4' | 'h5' | 'h6' | 'div' | 'p' | 'span';
+  /** Начертание шрифта */
+  weight?: 'normal' | 'medium' | 'bold';
+  /** Контент */
+  children: React.ReactNode;
+  /** Цвет */
+  color?: 'primary' | 'secondary' | 'accent';
+  /** Максимальное кол-во строк */
+  maxLines?: number;
+};
+
+const Text: React.FC<TextProps> = (props) => {
+  const { className, tag, children, maxLines } = props;
+
+  const Tag = tag ?? 'p';
+
+  const color = props.color ?? 'inherit';
+  const view = props.view ?? 'p-14';
```

Comment:
[+] можно использовать параметры по умолчанию, чтобы дополнительно не объявлять переменные
```suggestion
const Text: React.FC<TextProps> = ({
  className,
  tag: Tag = 'p',
  children,
  maxLines,
  color = 'inherit',
  view = 'p-14',
  weight = '',
}) => {
```
```